typescript/ws-protocol/README.md

```diff
@@ -99,28 +88,11 @@ async function main() {
 main().catch(console.error);
 ```
 
-## Example client
+## Client template
 
-The following script provides a simple example client which subscribes to messages on all channels that use the `json` encoding. To get started with the example client:
+The example client template below subscribes to messages on all channels that use the `json` encoding. 
```

Comment:
Might want to mention that the `simple-client` in the examples repo has an example of protobuf encoding support.